### PR TITLE
Translation zh docs

### PR DESCRIPTION
**Pull request by qianmoQ**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

Detailed description / Documentation draft:
...

> By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

> If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/en/engines/table-engines/integrations/kafka.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Optional parameters:
 -   `kafka_row_delimiter` — Delimiter character, which ends the message.
 -   `kafka_schema` — Parameter that must be used if the format requires a schema definition. For example, [Cap’n Proto](https://capnproto.org/) requires the path to the schema file and the name of the root `schema.capnp:Message` object.
 -   `kafka_num_consumers` — The number of consumers per table. Default: `1`. Specify more consumers if the throughput of one consumer is insufficient. The total number of consumers should not exceed the number of partitions in the topic, since only one consumer can be assigned per partition.
--   `kafka_max_block_size` — The maximum batch size (in messages) for poll (default: `max_block_size`).
+-   `kafka_max_block_size` — The maximum batch size (in messages) for poll (default: `max_block_size`, but not larger than current `max_block_size`.
```
Comment (mathalex) [inline docs/en/engines/table-engines/integrations/kafka.md:51]:
```suggestion
-   `kafka_max_block_size` — The maximum batch size (in messages) for poll (default: `max_block_size`, but not larger than current `max_block_size`).
```

File: docs/zh/engines/database-engines/postgresql.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+| DOUBLE           | [Float64](../../sql-reference/data-types/float.md)           |
+| DECIMAL, NUMERIC | [Decimal](../../sql-reference/data-types/decimal.md)       |
+| SMALLINT         | [Int16](../../sql-reference/data-types/int-uint.md)          |
+| INTEGER          | [Int32](../../sql-reference/data-types/int-uint.md)          |
+| BIGINT           | [Int64](../../sql-reference/data-types/int-uint.md)          |
+| SERIAL           | [UInt32](../../sql-reference/data-types/int-uint.md)         |
+| BIGSERIAL        | [UInt64](../../sql-reference/data-types/int-uint.md)         |
+| TEXT, CHAR       | [String](../../sql-reference/data-types/string.md)           |
+| INTEGER          | Nullable([Int32](../../sql-reference/data-types/int-uint.md))|
+| ARRAY            | [Array](../../sql-reference/data-types/array.md)             |
```
Comment (mathalex) [inline docs/zh/engines/database-engines/postgresql.md:45]: Fix the border (line 37) and add one more space at the end of each line in the second column as you did it before (due to line 44).

File: docs/zh/engines/database-engines/mysql.md [hunk 3]
```diff
@@ -10,67 +15,86 @@ MySQL引擎用于将远程的MySQL服务器中的表映射到ClickHouse中，并
...
+
+为了更好地兼容，您可以在SQL样式中设置全局变量，如`@@identifier`. 
 
-同时以上的所有类型都支持[Nullable](../../sql-reference/data-types/nullable.md)。
+支持这些变量：
+- `version`
+- `max_allowed_packet`
 
-## 使用示例 {#shi-yong-shi-li}
+!!! warning "Warning"
```
Comment (mathalex) [inline docs/zh/engines/database-engines/mysql.md:62]: Quoted words should be translated as the reader see them -> `!!! warning "(translation of warning)"`.

File: docs/zh/engines/database-engines/replicated.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+CREATE DATABASE testdb ENGINE = Replicated('zoo_path', 'shard_name', 'replica_name') [SETTINGS ...]
+```
+
+**引擎参数**
+
+-   `zoo_path` — ZooKeeper地址，同一个ZooKeeper路径对应同一个数据库。
+-   `shard_name` — 分片的名字。数据库副本按`shard_name`分组到分片中。
+-   `replica_name` — 副本的名字。同一分片的所有副本的副本名称必须不同。
+
+!!! note "Warning"
```
Comment (mathalex) [inline docs/zh/engines/database-engines/replicated.md:19]: Quoted words should be translated as the reader see them -> `!!! note "(translation of warning)"`.
Also, it is strange that in one place you have used warning and in another note with "Warning".

**Review by mathalex — COMMENTED** [review]
Something strange has happened... Submodules should not be changed in this PR...

**Comment by qianmoQ** [comment]
> Something strange has happened... Submodules should not be changed in this PR...

What should I do to fix it?

**Comment by qianmoQ** [comment]
> Something strange has happened... Submodules should not be changed in this PR...

When I was git pull, I saw that the record of these 160 files was delete

**Comment by qianmoQ** [comment]
Let me close it and do it again

**Pull request closed without merging**